source/nodejs/adaptivecards-designer/src/adaptivecards-designer.css

```diff
@@ -788,9 +788,10 @@
     border: 1px solid black;
 }
 
-.acd-toolbar-button:disabled {
+.acd-toolbar-button.acd-toolbar-button-disabled {
```

Comment:
This change will break apps that are customizing the existing styles. While I don't disagree that it might have been better to name the class as you have it here, I don't think this change is necessary.

Reply:
Using the `:disabled` state prevents focus.

Reply:
Oh sorry, my mistake - I didn't see the ":" and thought it was a "." You're introducing a new class.

Got it, looks good.